README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](http://aka.ms/dotnet-cli-eula).
```

Comment:
You don't need too, it's enabled, it's just the awful tool UI never tells you. Try it :) https://aka.ms/dotnet-cli-eula